artisan.md

```diff
@@ -497,6 +509,7 @@ If your command defines an option that accepts an array, you may pass an array o
         ]);
     });
 
+<a name="passing-boolean-values"></a>
 #### Passing Boolean Values
 
 If you need to specify the value of an option that does not accept string values, such as the `--force` flag on the `migrate:refresh` command, you should pass `true` or `false`:
```

Comment:
Is this just GitHub doing some weird highlighting or something? The text is highlighted like it's being deleted or something, but it isn't showing as lines deleted in the sidebar

Reply:
Screenshot in case it is just me

<img width="1224" alt="Screen Shot 2020-10-20 at 4 10 38 pm" src="https://user-images.githubusercontent.com/24803032/96542885-d16c8a80-12ee-11eb-8d5f-4e9d599492b3.png">

Reply:
🤷‍♂️ The rich-diff looks normal. Doesn't show any content missing. Maybe just something I've not seen on GitHub before. Ignore me.

Reply:
Yeah I noticed that as well. Seems to be a syntax highlighting glitch not directly related to this change. Perhaps there is some invalid Markdown elsewhere in the file throwing it off though...

Reply:
Yeah there's a long outstanding problem with Github breaking over some diffs. Feel free to ignore.